src/sentry/tasks/post_process.py

```diff
@@ -160,6 +160,7 @@ def post_process_group(event, is_new, is_regression, is_new_group_environment, *
         event.project._organization_cache = Organization.objects.get_from_cache(
             id=event.project.organization_id
         )
+        bind_organization_context(event.project._organization_cache)
```

Comment:
This _seems_ find to add, but I don't really if there is something missing as to why it shouldn't be okay, @dan (or anyone else relevant) any thoughts?

Reply:
For the cache, it won't hurt, but you could just pass `event.project.organization`. `_organization_cache` is just django magic.

```
In [3]: project = Project.objects.first()
In [4]: project._organization_cache
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-4-716d9694471d> in <module>()
----> 1 project._organization_cache

AttributeError: 'Project' object has no attribute '_organization_cache'

In [5]: project.organization
Out[5]: <Organization at 0x10bb33910: id=1L, owner_id=None, name=u'Sentry', slug=u'sentry'>

In [6]: project._organization_cache
Out[6]: <Organization at 0x10bb33910: id=1L, owner_id=None, name=u'Sentry', slug=u'sentry'>
```

I don't know if there's any reason you can't use `bind_organization_context` here.

Reply:
@wedamija My concern with using `event.project.organization` was not knowing if it would always be there. I figured this was safer since we are doing a `get_from_cache` for the org right before. I could assign it to an intermediate variable if that made things more clear.

Reply:
I mean it doesn't really matter either way, but since we just assigned the cache then `event.project.organization` is equivalent to `event.project._organization_cache`.

Reply:
Maybe I'm being overly paranoid, but I feel like we could just use `bind_organization_context` right before we start the new transaction https://github.com/getsentry/sentry/pull/20022/files#diff-f46f0c4b74ece0026caa075a343d2f17R135

We are adding this for a temporary (several months potentially) metric. Unless @scefali you were thinking this could be applied to all alerting, in that case maybe we can move it to https://github.com/getsentry/sentry/pull/20022/files#diff-8a40a9e4bd529742eed53f70103e6e63R180

Reply:
@MeredithAnya I want this tag for all transactions. Both the new one and `rule_processor_apply`. The second link didn't seem to go anywhere.

Reply:
oop thats awk, I tried to link right before the transaction in post_process https://github.com/getsentry/sentry/blob/master/src/sentry/tasks/post_process.py#L181

Reply:
@MeredithAnya Got it. I prefer to keep the `bind_organization_context` right next to where we are setting the organization so it's clear of the relationship between the two. Pretty sure functionally it will be the same as long as there is an `event.group_id`.